src/libinjector/injector.c

```diff
@@ -1521,20 +1534,14 @@ int injector_start_app(
         }
         else
         {
-            uint32_t err = 0;
-            const char* err_str = "<UNKNOWN>";
-            if (VMI_SUCCESS != drakvuf_get_last_error(drakvuf, 0, &err, &err_str))
-                err = -1;
-
             injector->result = INJECT_RESULT_ERROR_CODE;
-            injector->error_code = err;
-            injector->error_string = err_str;
             print_injection_info(format, file, injector);
         }
     }
 
     rc = injector->rc;
-    PRINT_DEBUG("Finished with injection. Ret: %i\n", rc);
+    PRINT_DEBUG("Finished with injection. Ret: %i. Error: %s(%d)\n", rc,
+                injector->error_string ?: "OK", injector->error_code);
```

Comment:
Are these changes intentional in `injector.c`? I just noticed them and they don't like like they belong in this PR

Reply:
They are not -- I didn't add that change myself. I'll see if I can figure out what happened there.

Reply:
I see that change in both our branches (your master and my json). It was introduced by zodeak on 5 Feb, so it must have been pulled in from the rebase. That would imply the left side of this diff shows the repo state prior to my branch, rather than the current state of master.

Reply:
OK, yes, it matches what's in the latest master. I recommend instead of merging master into your branch using `git rebase`, it usually results in cleaner results.